web_src/js/features/repo-common.ts

```diff
@@ -53,20 +53,34 @@ export function substituteRepoOpenWithUrl(tmpl: string, url: string): string {
 function initCloneSchemeUrlSelection(parent: Element) {
   const elCloneUrlInput = parent.querySelector<HTMLInputElement>('.repo-clone-url');
 
-  const tabSsh = parent.querySelector('.repo-clone-ssh');
-  const tabHttps = parent.querySelector('.repo-clone-https');
+  const tabHTTPS = parent.querySelector('.repo-clone-https');
+  const tabSSH = parent.querySelector('.repo-clone-ssh');
```

Comment:
It seems unnecessary change. In JS, there are usually "Url", "Https", "Ssh", but not "URL", "HTTPS", "SSH".